pktoken/pktoken.go

```diff
@@ -357,26 +384,18 @@ func (p *PKToken) VerifyCosSig(cosPk jwk.Key, alg jwa.KeyAlgorithm) error {
 	return nil
 }
 
-func (p *PKToken) Verify(msg string, sig []byte) error {
-	jwsSig, err := jws.Parse(sig)
+func (p *PKToken) Hash() (string, error) {
+	// get pktoken bytes
+	pktJson, err := p.ToJSON()
 	if err != nil {
-		return err
-	}
-
-	if msg != string(jwsSig.Payload()) {
-		return fmt.Errorf("Message does not match signed message")
-	}
-
-	alg, _, upk, err := p.GetCicValues()
-	if err != nil {
-		return err
+		return "", err
 	}
 
-	_, err = jws.Verify(sig, jws.WithKey(alg, upk))
+	hasher := sha256.New()
```

Comment:
It isn't written down anywhere, but OpenPubkey should use SHA-3 except when required to use SHA-256 by the JWS standard.